server/portal/settings/settings.py

```diff
@@ -726,9 +725,13 @@ def gettext(s): return s  # noqa:E731
 """
 SETTINGS: RECAPTCHA
 """
-RECAPTCHA_SECRET_KEY = settings_secret._RECAPTCHA_SECRET_KEY
-RECAPTCHA_SITE_KEY =  settings_secret._RECAPTCHA_SITE_KEY
-
+if hasattr(settings_secret, '_RECAPTCHA_SECRET_KEY'):
```

Comment:
Instead of using if/else here, something like below may be a little cleaner

`RECAPTCHA_SECRET_KEY = getattr(settings_secret, '_RECAPTCHA_SECRET_KEY', 'no_key')`
`RECAPTCHA_SITE_KEY = getattr(settings_secret, '_RECAPTCHA_SITE_KEY', 'no_key')`